worlds/.soccer1.wbproj

```diff
@@ -0,0 +1,9 @@
+Webots Project File version R2023a
```

Comment:
Do we need these `.soccerX.wbproj` files? What are they used for?

Reply:
Hello @Adman,

Sorry, we were not aware that there were those files in the folder. We assume that it may have been a file that was created by the Webots program when we were trying to making our Webots world with the R2023a version. Our guess is that it would not be a problem to simply delete those files.

On a side note, we originally created the world to work with the R2023a version rather than the R2022b version of Webots. However, due to issues with obtaining the correct values for the ball direction (the issue was only present in the Mac version of Webots but not the Windows version, for some reason), we ended up changing our code to work with the R2022b version. As a result, there may likely be unneccessary files created by the R2023a version of Webot's program.

Team Reset

Reply:
Thank you.